trillian/integration/ct_integration_test.cfg

```diff
@@ -12,7 +12,7 @@ config {
 		}
 	}
 	max_merge_delay_sec: 86400
-	expected_merge_delay_sec: 7200
+	expected_merge_delay_sec: 120
```

Comment:
Could you add comments indicating that 120s is unrealistically fast for a real log, but this OK for development/testing.

Reply:
Can I do this in a text proto file? I tried appending "// blah", but it says:
> Failed to read config: failed to parse log config: line 15: unknown field name "/" in configpb.LogConfig

Reply:
http://lmgtfy.com/?q=text+proto+comment

Reply:
Ah, "# blah" works.

Reply:
Done.